packages/dashboard/src/components/template/you-might-like/you-might-like.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import YouMightLikeCard from './you-might-like-card';
+
+const trader_types = [
```

Comment:
- Since this is defined outside of the component it'll only get executed once, so the `localize` will stop working when the user changes their language (unless they refresh).
- Don't forget to `localize` the `title` keys.

Reply:
What happened to the `localize`? 🤔

Reply:
@aaron-binary  Seems like she placed it directly in the `you-might-like-card` component as it takes in the titles and subtitles.

Reply:
@sean-binary the titles in this file needs to be localized as well

Reply:
@akmal-binary Oh right, we'll need to extract them in the first place 😅

Reply:
since this is only for template, these texts are just for example, when starting the implementation, please ensure to take note aaron's note